### ISSUE TITLE
Setup: Gather participant inforrmation

### ISSUE THREAD

**Issue by LIN810116** [ABI-CTT-Group/digital-twin-platform#20]
- Request name, UPI & email address
- Request information on the dataset they will contribute to the platform (type of data etc)

**Comment by PrasadBabarendaGamage** [comment]
@LIN810116 Participant information has been added to the bottom of https://docs.google.com/document/d/172BNK7_bteDIrFy1kgQlZIlijMZh4O2e170HRWVFYcc/edit

**Comment by LIN810116** [comment]
Hi @PrasadBabarendaGamage ,

Thank you for gathering the participant information.

I just noticed one problem that many of the participants are using the `auckland.ac.nz` email domain, which is based on Microsoft Outlook and not supported by our current Gen3 setup.  Our current Gen3 only supports Google login (`aucklanduni.ac.nz` would work)

One good thing is that I figured out a way to set up a default user (this has to disable Google login) so that users do not necessarily need to log into the Gen3 portal to generate the API key.

Gen3 (portal & API) will become accessible if they: 
1. ssh -L 80:localhost:80 UPI:VM_IP
2. then they can access Gen3 via localhost:80
